cvmfs/cvmfsdirtab.sh

```diff
@@ -25,30 +25,28 @@ for cmsdir in "$@" ; do
   done
   if [ -d ${CVMFS_DIR}/$cmsdir/spack ]; then
     echo "/${cmsdir}/spack/share"
-    echo "/${cmsdir}/spack/share/data-l1trigger-l1tmuon"
-    echo "/${cmsdir}/spack/share/data-generatorinterface-evtgeninterface"
-    echo "/${cmsdir}/spack/share/data-magneticfield-interpolation/*"
+    echo "/${cmsdir}/spack/share/cms/data-l1trigger-l1tmuon"
+    echo "/${cmsdir}/spack/share/cms/data-generatorinterface-evtgeninterface"
+    echo "/${cmsdir}/spack/share/cms/data-magneticfield-interpolation/*"
```

Comment:
what about cleanup of these extra `cms` paths?

Reply:
I will take care of them in another PR, still checking.